src/main/java/catan/controllers/PlayController.java

```diff
@@ -29,7 +32,10 @@ public void buildRoad(@RequestParam(value = "token", required = false) String to
                           @RequestParam("edgeId") String edgeId) throws AuthenticationException, GameException, PlayException {
         UserBean user = authenticationService.authenticateUserByToken(token);
 
-        playService.buildRoad(user, gameId, edgeId);
+        HashMap<String, String> params = new HashMap<String, String>();
```

Comment:
TBD: do we need a HashMap variable here or we can push into processAction String param instead?
For now we push only one parameter into controllers and it has a String type